code/modules/food_and_drinks/drinks/drinks/breakawayflask.dm

```diff
@@ -0,0 +1,81 @@
+/obj/item/reagent_containers/food/drinks/breakawayflask
+	name = "breakaway flask"
+	desc = "A special flask designed to stabilize trick wines and shatter violently on contact."
+	icon_state = "breakawayflask"
+	item_state = "breakawayflask"
+	gulp_size = 25
+	amount_per_transfer_from_this = 25
+	volume = 50
+	throwforce = 20
```

Comment:
Turn down the damage to 10, and maybe increase the size to WEIGHT_CLASS_SMALL? 20 damage is comparable to a spear.